code/modules/projectiles/ammo_datums.dm

```diff
@@ -1241,6 +1241,88 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	sundering = 1
 	damage_falloff = 0.1
 
+/datum/ammo/bullet/smarttargetrifle
+	name = "smart marksman bullet"
+	bullet_color = COLOR_SOFT_RED //Red bullets to indicate friendly fire restriction
+	hud_state = "smartgun"
+	hud_state_empty = "smartgun_empty"
+	flags_ammo_behavior = AMMO_BALLISTIC|AMMO_SUNDERING
+	damage = 40
+	penetration = 30
+	sundering = 5
+	shell_speed = 4
+	damage_falloff = 0.5
+	accurate_range = 25
+	accurate_range_min = 3
```

Comment:
This is essentially pointless. At 2 tiles you're in pb range, so a min range of 3 means ONLY at 3 tiles away you get slightly less accuracy.

Reply:
I could probably just remove it with that in consideration, honestly.